### ISSUE TITLE
Japanese characters not displayed in the integrated console when paste it.

### ISSUE THREAD

**Issue by mkht** [PowerShell/vscode-powershell#1036]
### Issue Description
When I paste japanese chars to the integrated console, pasted chars not displayed.

![code](https://user-images.githubusercontent.com/25482642/30752635-df531906-9ff7-11e7-9010-ebf175726736.gif)

#### How to repro
1. Copy japanese text. (e.g 'こんにちは')
1. Paste it to the integrated console.

### System Details
- OS Version: Windows 10 Pro x64 Japanese (build 15063.608)
```
> code -v
1.16.1
27492b6bf3acb0775d82d2f87b25a93490673c6d

> $pseditor.EditorServicesVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
1      4      1      0

> $PSVersionTable
PSVersion                      5.1.15063.608
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.15063.608
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Attached Logs
[logs.zip](https://github.com/PowerShell/vscode-powershell/files/1325030/logs.zip)

**Comment by rkeithhill** [comment]
This one is weird.  If you press <kbd>Enter</kbd> you'd see the characters displayed.  If you paste a single char at a time, you will see that char.  But when you paste the next char, the previous char disappears.

**Comment by rkeithhill** [comment]
Update: I think this is related to - https://github.com/Microsoft/vscode/issues/19665#issuecomment-287248500

I can copy other three byte UTF8 sequence characters into the console with no problem e.g. `↕` is the three byte seq `E2 86 95` whereas the `こ` is `E3 81 93`. However the former is a one-column char and the latter is two-column char.

**Comment by SydneyhSmith** [comment]
Closing as we have resolved this-- see #1761
